docs/notebooks/Common_Gotchas_in_JAX.ipynb

```diff
@@ -1636,7 +1636,7 @@
     "\\hline\n",
     "\\end{array}\n",
     "$$\n",
-    "<center>$\\ast$ = argument-__value__-independent loop condition - unrolls the loop </center>"
+    "<center>\n\n$\\ast$ = argument-__value__-independent loop condition - unrolls the loop\n\n</center>"
```

Comment:
```suggestion
    "<center>\\n\\n$\\ast$ = argument-__value__-independent loop condition - unrolls the loop\\n\\n</center>"
```

Reply:
Adding these additional backslashes to the notebook actually seems to break the generated html docs again.